plugins/org.python.pydev.core/src/org/python/pydev/core/docutils/PySelection.java

```diff
@@ -936,50 +939,63 @@ public static String getTextForCompletionInConsole(IDocument document, int docum
      *
      * @return the activation token and the qualifier.
      */
-    public static ActivationTokenAndQual getActivationTokenAndQual(IDocument doc, int documentOffset,
+    public static ActivationTokenAndQualifier getActivationTokenAndQualifier(IDocument doc, int documentOffset,
             boolean getFullQualifier, boolean handleForCalltips) {
         boolean changedForCalltip = false;
         boolean alreadyHasParams = false; //only useful if we're in a calltip
         int parOffset = -1;
         boolean isInMethodKeywordParam = false;
         int offsetForKeywordParam = -1;
-
         int foundCalltipOffset = -1;
+        boolean isInKeytip = false;
```

Comment:
The code here should be reverted since we shouldn't be using the `isInKeyTip` anymore.